src/script_functions.c

```diff
@@ -1689,15 +1625,11 @@ int c_get_window_type(lua_State *lua)
  */
 int c_get_class_instance_name(lua_State *lua)
 {
-#ifdef HAVE_GTK3
-	int top = lua_gettop(lua);
-
-	if (top != 0) {
-		luaL_error(lua, "get_window_class_instance_name: %s",
-		           no_indata_expected_error);
+	if (!check_param_count(lua, "get_class_instance_name", 0)) {
 		return 0;
 	}
 
+#ifdef HAVE_GTK3
```

Comment:
I wasn't sure if this should stay inside the ifdef or be applied even if the function is a noop.

Reply:
Check anyway seems like the right choice to me.